webapp/src/components/LogoSign/index.tsx

```diff
@@ -74,7 +74,7 @@ function Logo() {
 
   return (
     <Tooltip title="Totoro Admin Dashboard" arrow>
-      <LogoWrapper to="/dashboards/statistiques">
+      <LogoWrapper to="/dashboards/stats">
```

Comment:
```suggestion
      <LogoWrapper to={DASHBOARD_STATS_PATH}>
```

see the comment above